README.md

```diff
@@ -246,6 +246,8 @@ Usage of tg_history_dumper:
   -dump-sessions string
         enable active sessions dump, use 'write' to enable dump, overriders config.dump_sessions
   -list-chats
+        Logout from current session
```

Comment:
Argument description must be below argument name.

And please also add this description to `README_zh.md` (it's ok to use English unless you know how to translate it correctly).